services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeServiceImpl.kt

```diff
@@ -112,9 +112,6 @@ open class HankeServiceImpl(
      * @return a new Hanke instance with the added and possibly modified values.
      */
     override fun createHanke(hanke: Hanke): Hanke {
-        // TODO: Only create that hanke-tunnus if a specific set of fields are non-empty/set.
```

Comment:
This is actually quite relevant note; should be mentioned either here, or in confluence (future notes or wherever). (That is, the main point is, I guess, that the service does not currently do any validation about the incoming data.)

Reply:
Validation is on Controller level. Maybe here would also need some validation but that is not done atm. Put back the TODO.